src/style/editor.ts

```diff
@@ -1,9 +1,12 @@
-.rdg-editor-container {
+import { css } from '@linaria/core';
+
+const editorContainer = css`
```

Comment:
For example this style can be move to `EditorContainer.tsx` and the next to `TextEditor`